### PR TITLE
Add tooltip to GameTableController

### LINKED ISSUE #804: Play -> Custom Games game tooltip to large
play_custom_games_game_tooltip_to_large

### LINKED ISSUE #1338: Tooltip with mods for coop games (table view)
Related to #1014
Add Tooltip (same as used in tiles view) to ~currently selected~ any row in `GamesTableController`.

### PR DESCRIPTION
**Pull request by Katharsas**
Same tooltip controller as used in GameTileController.
Fixes #1338
Fixes #804 

May need some help with tests if those are wanted and if its even possible to test thist.

### CODE REVIEW

**Comment by 1-alex98** [comment]
Can u post a screen shot so I can imagine what u are doing here?

**Comment by Katharsas** [comment]
coop_tooltip
(Same for table view of custom games. I can restrict it to only show hen mouse hovers over selected row, but right now it shows tooltips for any row hen hovered over).

**Comment by 1-alex98** [comment]
This is now also the case in the table of normal games right....

**Comment by 1-alex98** [comment]
I am on discord and we can try to figure out how to test it...

**Comment by Katharsas** [comment]
yeah thats what i meant.

**Review by Brutus5000 — COMMENTED** [review]
I'm not convinced by this change yet. Why do we need to reload the tooltip every time from the fxml? I guess this is pretty expensive. Why not store one loaded controller and just update the displayed game?

**Comment by 1-alex98** [comment]
We fixed this along with many other things concerning the tool tip a second ago

File: src/main/java/com/faforever/client/game/GamesTableController.java [hunk 1]
```diff
@@ -103,7 +110,7 @@ public void initializeGameTable(ObservableList<Game> games, Function<String, Str
 
     applyLastSorting(gamesTable);
     gamesTable.setOnSort(this::onColumnSorted);
-
+
```
Comment (1-alex98) [inline src/main/java/com/faforever/client/game/GamesTableController.java]: ?

File: src/test/java/com/faforever/client/game/GamesTableControllerTest.java [hunk 2]
```diff
@@ -50,20 +65,26 @@ public void setUp() throws Exception {
 
   @Test
   public void test() throws Exception {
-    instance.initializeGameTable(FXCollections.observableArrayList(
-        GameBuilder.create().defaultValues().get(),
-        GameBuilder.create().defaultValues().state(GameStatus.CLOSED).get()
-    ));
+    Platform.runLater(() -> {
```
Comment (1-alex98) [inline src/test/java/com/faforever/client/game/GamesTableControllerTest.java:66]: Use WaitForAsyncUtils.executeAsyncFX something

**Review by 1-alex98 — CHANGES_REQUESTED** [review]

File: src/test/java/com/faforever/client/game/GamesTableControllerTest.java (hunk 2 above)
Comment (1-alex98) [reply src/test/java/com/faforever/client/game/GamesTableControllerTest.java:66]: and then get()

**Comment by Brutus5000** [comment]
Looks better to me now, but I'm not deep into the JavaFX stuff to actually approve this. It's up to @axel1200 now.

**Review by 1-alex98 — APPROVED** [review]

**Pull request merged**